mentor-me/src/App.js

```diff
@@ -8,7 +8,7 @@ import Profile from './components/Profile/Profile'
 import Login from './components/Login/Login'
 import Signup from './components/SignUp/Signup'
 
-function App() {
+export default function App() {
```

Comment:
Since you already have `export default` at the bottom of the file, you can remove it here.